reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,250 @@
+
+To make the transition from the **ISE** to **VSCode** smoother, a new option **Enable ISE Mode** is
+available in the the Command Palette to switch **VSCode** into an **ISE** style layout. By
+transitioning to **VSCode** and the new **ISE** layout, you will continue to get all the new
+features and capabilities of PowerShell in a familiar scripting tool.
+
+To switch between the default layout and the new ISE layout, press;
+
+<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>
+
+Type and select: **PowerShell:Enable ISE Mode**
+
+To set the layout to the original VSCode layout, open the Command Palette, type and select
+**PowerShell: Disable ISE Mode (restore to defaults)**
+
+For details about customizing the VSCode layout to ISE, see
+[How to Replicate the ISE Experience in Visual Studio Code](https://docs.microsoft.com/powershell/scripting/components/vscode/how-to-replicate-the-ise-experience-in-vscode?view=powershell-7)
+
+> [!NOTE]
+> The PowerShell Team and partners are actively working to improve the scripting experience in the
+> PowerShell extension for VSCode. We currently have no plans to update the ISE with new features
+> going forward. In the latest versions of Windows 10 and Windows Server, the ISE is now a
+> user-uninstallable feature, but this does not signal any plans to make it unavailable in the  future.
+
+## PowerShell SSH Remoting
+
+In previous versions of Windows PowerShell, Remoting was handled through **WinRM** for connection
+negotiation and data transport. To support cross-platform needs for Windows and Linux, **SSH-based**
+remoting is now available and allows true multiplatform PowerShell Remoting.
+
+SSH remoting creates a PowerShell host process on the target computer as an SSH subsystem. For
+details and examples on setting up Windows or Linux for SSH-based remoting, see:
+[PowerShell Remoting over SSH](https://docs.microsoft.com/powershell/scripting/learn/remoting/ssh-remoting-in-powershell-core?view=powershell-7).
+
+The `New-PSSession`, `Enter-PSSession`, and `Invoke-Command` cmdlets now have a new parameter set to
+support this new remoting connection.
+
+```powershell
+[-HostName <string>]  [-UserName <string>]  [-KeyFilePath <string>]
+```
+
+To create a remote session, specify the target computer with the **HostName** parameter and provide
+the user name with **UserName**. When running the cmdlets interactively, you're prompted for a
+password.
+
+```powershell
+Enter-PSSession -HostName <Computer> -UserName <Username>
+```
+
+Alternatively, When using the **HostName** parameter, provide the username information followed by
+the **@** sign, followed by the computer name.
+
+```powershell
+Enter-PSSession -HostName <Username>@<Computer>
+```
+
+You may setup **SSH** key authentication using a private key file with the KeyFilePath parameter.
+For more information, see;
+[OpenSSH Key Management](https://docs.microsoft.com/windows-server/administration/openssh/openssh_keymanagement).
```

Comment:
The article ends kind of abruptly. Should we add a "call to action" or "next steps".